akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -693,5 +695,28 @@ public void illustrateOrderingAndParallelismOfMapAsyncUnordered() throws Excepti
     };
   }
 
+  @Test
+  public void illustrateSourceQueue() throws Exception {
+    new TestKit(system) {
+      {
+        //#source-queue
+        int bufferSize = 5;
+        int elementsToProcess = 3;
+
+        SourceQueueWithComplete<Integer> sourceQueue = Source.<Integer>queue(bufferSize, OverflowStrategy.backpressure())
+            .throttle(elementsToProcess, Duration.ofSeconds(3), elementsToProcess, ThrottleMode.shaping())
```

Comment:
use the simplified signature: `.throttle(elementsToProcess, Duration.ofSeconds(3))

Reply:
thanks! great that we have the simpler one